ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -35,6 +35,9 @@ ethernet_interfaces:
 {%                     endif %}
     type: switched
     shutdown: false
+{%                     if adapter_settings.enabled is arista.avd.defined %}
+    shutdown: {{ not adapter_settings.enabled }}
+{%                     endif %}
```

Comment:
Please update this in a positive stance, and avoid generating duplicate keys

Old syntax:
```jinja
    shutdown: false
{%                     if adapter_settings.enabled is arista.avd.defined %}
    shutdown: {{ not adapter_settings.enabled }}
{%                     endif %}
```
Proposed new syntax:
```jinja
{%                     if adapter_settings.enabled is arista.avd.defined(false) %}
    shutdown: true
{%                     else %}
    shutdown: false
{%                     endif %}
```

Reply:
Thank you for the hint to the documentation. And also thank you for the code style change request. Both done.